sdcm/nemesis.py

```diff
@@ -4946,3 +4978,32 @@ def __init__(self, *args, **kwargs):
         super(SisyphusMonkey, self).__init__(*args, **kwargs)  # pylint: disable=bad-super-call
         self.build_list_of_disruptions_to_execute(nemesis_selector=['free_tier_set'])
         self.shuffle_list_of_disruptions()
+
+
+class SlaIncreaseSharesDuringLoad(Nemesis):
+    disruptive = False
+    sla = True
+
+    def disrupt(self):
+        self.disrupt_sla_increase_shares_during_load()
+
+
+class SlaDecreaseSharesDuringLoad(Nemesis):
+    disruptive = False
+    sla = True
+
+    def disrupt(self):
+        self.disrupt_sla_decrease_shares_during_load()
```

Comment:
I'm afraid that we will have inflation of nemesis with this feature.
(I know that there are planned at least 4-6 more).

The problem with this is that for non-enterprise runs we may still choose them, and even if we skip them, we "lose" the time that it took (status + interval + status).
In this case, the chance to get a suitable nemesis is getting lower.

I can think of two solutions to that:
1. Add a "enterprise" property to all the enterprise nemesis and add it to the nemesis_selector.
2. The call for disruption to be changed and if we chose a nemesis that was skipped, we shall choose another one immediately.

Reply:
I think option two is much better, and how I thought we should use skip when I implemented it. (not sure I we got the situation as it today...)

Reply:
I saw your PR with adding back the "raise", but I don't understand if/how it handles this.

Reply:
It's doing exactly what's written in number 2, it raise the exception up to the main nemesis loop, which then do a wait=0, and skip to the next nemesis straight away 
some where between 4.5-4.6, the logic broke, and someone had remove that raise.

Reply:
found the issue and fixed in a different PR